src/main/scala/com/typesafe/sbt/digest/SbtDigest.scala

```diff
@@ -104,24 +104,27 @@ object SbtDigest extends AutoPlugin {
       include: FileFilter,
       exclude: FileFilter,
       targetDir: File,
-      indexPathOpt: Option[String],
+      indexFileOpt: Option[File],
       indexWriter: Map[String, String] => String
     ): Seq[PathMapping] = {
       val processed = process(mappings, algorithms, include, exclude, targetDir)
-      val digested = indexPathOpt.fold(processed) { indexPath =>
+      val digested = indexFileOpt.fold(processed) { indexFile =>
         // generate an index file and add to digested outputs
         // replace any existing file at the index path
-        val filtered = processed filterNot (_.originalPath == indexPath)
+        val filtered = processed filterNot (_.originalPath == indexFile.toString)
```

Comment:
Is `indexFile.toString` the same as the previous `indexPath`? To be the same, won't it need to be a path relative to the target directory?